bookwyrm/templates/settings/schedules.html

```diff
@@ -0,0 +1,116 @@
+{% extends 'settings/layout.html' %}
+{% load i18n %}
+{% load humanize %}
+{% load utilities %}
+
+{% block title %}
+{% trans "Scheduled tasks" %}
+{% endblock %}
+
+{% block header %}
+{% trans "Scheduled tasks" %}
+{% endblock %}
+
+{% block panel %}
+
+<div class="block content">
+    <h3>{% trans "Tasks" %}</h3>
+    <div class="table-container">
+        <table class="table is-striped is-fullwidth">
+            <tr>
+                <th>
+                    {% trans "Name" %}
```

Comment:
A nice-to-have (possibly in a subsequent PR) would be to link the task name to some sort of explanation of the purpose of the task and/or what it does.

Reply:
Periodic tasks have a `description` field which we're currently not using, but presumably would be perfect for this